helmfile.d/snippets/defaults.yaml

```diff
@@ -179,9 +179,22 @@ environments:
             enabled: false
             logLevel: info
         backup:
-          platformSchedule:
-            enabled: false
-          teamSchedule:
+          platform:
+            gitea:
+              enabled: false
```

Comment:
If PVC's and PV's are kept even after disabling of a given app, then the user would have to manually delete these resources as a prerequisite to enabling backup for just a single app. It would be biting off more than the user can chew. I'm definitely on the disabled-by-default side here.